src/appleseed/foundation/meta/tests/test_imagefilewriter.cpp

```diff
@@ -0,0 +1,219 @@
+// appleseed.foundation headers.
+#include "foundation/image/color.h"
+#include "foundation/image/colorspace.h"
+#include "foundation/image/genericimagefilereader.h"
+#include "foundation/image/genericimagefilewriter.h"
+#include "foundation/image/image.h"
+#include "foundation/image/imageattributes.h"
+#include "foundation/image/pixel.h"
+#include "foundation/math/scalar.h"
+#include "foundation/utility/iostreamop.h"
+#include "foundation/utility/test.h"
+
+// Standard headers.
+#include <cassert>
+#include <cmath>
+#include <cstddef>
+#include <cstring>
+#include <exception>
+#include <memory>
+
+using namespace foundation;
+
+/*
+// Default export formats:
+//
+// OpenEXR   .exr          4-channel   16-bit (half)                            Linear
+// RGBE      .hdr          3-channel   32-bit (8-bit RGB + shared exponent)     Linear
+// TIFF      .tiff/.tif    4-channel   16-bit (std::uint16_t)                   Linear
+// BMP       .bmp          4-channel    8-bit (std::uint8_t)                    sRGB
+// PNG       .png          4-channel    8-bit (std::uint8_t)                    sRGB
+// JPEG      .jpg/.jpe/    3-channel    8-bit (std::uint8_t)                    sRGB
+//           .jpeg/.jif/
+//           .jfif/.jfi
+*/
+
+TEST_SUITE(Foundation_Image_ImageFileWriter)
+{
+    bool write_image(const std::string ImageFilePath)
```

Comment:
`write_image` should not return a `bool`. 

args and variables names should be formatted as such : `image_file_path`